website/content/en/preview/getting-started/migrating-from-cas/_index.md

```diff
@@ -112,14 +122,14 @@ Replace the nodegroup value with your `${NODEGROUP}`
 
```

Comment:
Could you change L110 to say `- ${NODEGROUP}` to be more specific? this way it fails if they forget to change it. 

Same with L145